README.md

```diff
@@ -3,12 +3,15 @@
 
 # aws-ssm-env
 
-AWS EC2 Parameter Storeから取得したパラメータを環境変数として設定します。  
+Set parameters acquired from `AWS EC2 Parameter Store` as environment variables.  
```

Comment:
This sentence lacks subject.
It's better to add subject like `aws-ssm-env` or `This tool` etc.

For example)

> This tool sets parameters aquired from [AWS EC2 Parameter Store](https://docs.aws.amazon.com/systems-manager/latest/userguide/systems-manager-paramstore.html) as environment variables.